ipython_widgets/static/widgets/js/widget.js

```diff
@@ -524,10 +522,10 @@ define(["nbextensions/widgets/widgets/js/manager",
             WidgetView.__super__.remove.apply(this, arguments);
             this.trigger('remove');
         }
-    });
+    };
 
 
-    var DOMWidgetView = WidgetView.extend({
```

Comment:
Sorry, see my comment below.  I thought you were missing the DOMWidgetView definition.